2024-02-01 23:37:50,138 INFO    MainThread:3921 [wandb_setup.py:_flush():76] Current SDK version is 0.16.1
2024-02-01 23:37:50,138 INFO    MainThread:3921 [wandb_setup.py:_flush():76] Configure stats pid to 3921
2024-02-01 23:37:50,138 INFO    MainThread:3921 [wandb_setup.py:_flush():76] Loading settings from /Users/songzeyang/.config/wandb/settings
2024-02-01 23:37:50,138 INFO    MainThread:3921 [wandb_setup.py:_flush():76] Loading settings from /Users/songzeyang/Documents/GitHub/Data_Distill_with_NC/wandb/settings
2024-02-01 23:37:50,138 INFO    MainThread:3921 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-01 23:37:50,138 INFO    MainThread:3921 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-01 23:37:50,138 INFO    MainThread:3921 [wandb_init.py:_log_setup():524] Logging user logs to /Users/songzeyang/Documents/GitHub/Data_Distill_with_NC/wandb/run-20240201_233750-9ceu1i1q/logs/debug.log
2024-02-01 23:37:50,138 INFO    MainThread:3921 [wandb_init.py:_log_setup():525] Logging internal logs to /Users/songzeyang/Documents/GitHub/Data_Distill_with_NC/wandb/run-20240201_233750-9ceu1i1q/logs/debug-internal.log
2024-02-01 23:37:50,138 INFO    MainThread:3921 [wandb_init.py:_jupyter_setup():470] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x161ad01d0>
2024-02-01 23:37:50,138 INFO    MainThread:3921 [wandb_init.py:init():564] calling init triggers
2024-02-01 23:37:50,138 INFO    MainThread:3921 [wandb_init.py:init():571] wandb.init called with sweep_config: {}
config: {'model': 'resnet18', 'bias': True, 'ETF_fc': True, 'fixdim': 0, 'SOTA': True, 'width': 1024, 'depth': 6, 'gpu_id': 0, 'seed': 6, 'use_cudnn': True, 'dataset': 'cifar10', 'data_dir': '~/data', 'uid': 'tmp', 'force': True, 'epochs': 200, 'batch_size': 2048, 'loss': 'CrossEntropy', 'sample_size': None, 'lr': 0.001, 'optimizer': 'Adam', 'history_size': 10, 'device': 'cuda'}
2024-02-01 23:37:50,138 INFO    MainThread:3921 [wandb_init.py:init():614] starting backend
2024-02-01 23:37:50,138 INFO    MainThread:3921 [wandb_init.py:init():618] setting up manager
2024-02-01 23:37:50,139 INFO    MainThread:3921 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-02-01 23:37:50,140 INFO    MainThread:3921 [wandb_init.py:init():624] backend started and connected
2024-02-01 23:37:50,147 INFO    MainThread:3921 [wandb_run.py:_label_probe_notebook():1294] probe notebook
2024-02-01 23:37:50,147 INFO    MainThread:3921 [wandb_run.py:_label_probe_notebook():1304] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-02-01 23:37:50,147 INFO    MainThread:3921 [wandb_init.py:init():716] updated telemetry
2024-02-01 23:37:50,177 INFO    MainThread:3921 [wandb_init.py:init():749] communicating run to backend with 90.0 second timeout
2024-02-01 23:37:50,727 INFO    MainThread:3921 [wandb_run.py:_on_init():2254] communicating current version
2024-02-01 23:37:50,797 INFO    MainThread:3921 [wandb_run.py:_on_init():2263] got version response upgrade_message: "wandb version 0.16.2 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-02-01 23:37:50,797 INFO    MainThread:3921 [wandb_init.py:init():800] starting run threads in backend
2024-02-01 23:37:53,192 WARNING MainThread:3921 [wandb_init.py:init():1192] interrupted
Traceback (most recent call last):
  File "/Users/songzeyang/anaconda3/lib/python3.11/site-packages/wandb/sdk/wandb_init.py", line 1170, in init
    run = wi.init()
          ^^^^^^^^^
  File "/Users/songzeyang/anaconda3/lib/python3.11/site-packages/wandb/sdk/wandb_init.py", line 811, in init
    run_start_result = run_start_handle.wait(timeout=30)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/songzeyang/anaconda3/lib/python3.11/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/songzeyang/anaconda3/lib/python3.11/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/songzeyang/anaconda3/lib/python3.11/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/songzeyang/anaconda3/lib/python3.11/threading.py", line 622, in wait
    signaled = self._cond.wait(timeout)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/songzeyang/anaconda3/lib/python3.11/threading.py", line 324, in wait
    gotit = waiter.acquire(True, timeout)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt
